spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -90,7 +90,9 @@
         "spark",
         SparkInterpreter.class.getName(),
         new InterpreterPropertyBuilder()
-            .add("spark.app.name", "Zeppelin", "The name of spark application.")
+            .add("spark.app.name",
```

Comment:
yeah, I think there are two different users here.  One is a site admin and they need to be able to set reasonable defaults for their environment (spark master, spark app name, etc.).  Then there are regular users who may want to override the defaults once they have a better understanding of zeppelin, spark and other interpreters.

So I did start a conversation on users, that's what lead to this PR.  I guess I can take a different tact, but ultimately I need to solve my problem now.  I'll have to maintain a fork until there is some consensus I guess.